components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -44,11 +43,11 @@ bool Spotify_FindCode(char *Response, uint16_t SizeRes)
 }
 
 /**
- * @brief This function handles the first HTTPS request to Spotify and redirects the user to the authorization page.
+ * @brief This function handles the first HTTPS request to Service and redirects the user to the authorization page.
  * @param[in] req The HTTP request object.
  * @return Returns ESP_OK if the request is processed successfully.
  */
-static esp_err_t Spotify_RequestDataAccess(httpd_req_t *HttpdRequest)
+static esp_err_t RequestDataAccess(httpd_req_t *HttpdRequest)
```

Comment:
hardcoded value in L60

Reply:
I'm not sure if `302` is standard value or customizable status that should be configurable! (L63)

Reply:
_[302 Found](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302)

    This response code means that the URI of requested resource has been changed temporarily. Further changes in the URI might be made in the future. Therefore, this same URI should be used by the client in future requests._
it looks fine I think!